### PR TITLE
R4 ChargeItem: Supporting Information field changes

### PR DESCRIPTION
**Pull request by krvicky31**
Description
----
fhir.cerner.com changes for ServiceRequest (OrderID) in chargeItem responses for Read, Search and Write

PR Checklist
----
- [x] Screenshot(s) of changes attached before changes merged.
Screenshot 2022-01-11 at 12 26 10 PM
Screenshot 2022-01-25 at 12 48 13 AM
Screenshot 2022-01-25 at 12 48 44 AM
- [ ] Screenshot(s) of changes attached after changes merged and published.

### CODE REVIEW

**Review by SudhirPai — APPROVED** [review]

**Review by BhavyaAcharya — APPROVED** [review]

File: lib/resources/example_json/r4_examples_charge_item.rb [hunk 1]
```diff
@@ -237,6 +237,11 @@ module Resources
         {
           'reference': 'Account/C98900463'
         }
+      ],
```
Comment (alexbeckner1) [inline lib/resources/example_json/r4_examples_charge_item.rb:240]: If it's not too late, we could replace the entire Read example from chargeitem 292870306 to 321088041 which has an actual serviceRequest reference 
`{
    "resourceType": "ChargeItem",
    "id": "321088041",
    "meta": {
        "versionId": "1",
        "lastUpdated": "2022-01-21T14:15:11Z"
    },
    "text": {
        "status": "generated",
        "div": "<div xmlns=\"http://www.w3.org/1999/xhtml\"><p><b>Charge Item</b></p><p><b>Status</b>: Billable</p><p><b>Patient</b>: 12768550</p><p><b>Encounter</b>: 97965088</p><p><b>Code</b>: 1010222</p><p><b>Occurrence</b>: Jan 21, 2022  1:54 P.M. UTC</p></div>"
    },
    "extension": [
        {
            "valueMoney": {
                "value": 1,
                "currency": "USD"
            },
            "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/unit-price"
        },
        {
            "valueMoney": {
                "value": 1,
                "currency": "USD"
            },
            "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/net-price"
        },
        {
            "valueReference": {
                "reference": "ChargeItem/321088013"
            },
            "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/replacing"
        },
        {
            "valueReference": {
                "reference": "Location/2572643903",
                "display": "CITY-NU02"
            },
            "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/performing-location"
        },
        {
            "valueString": "Blood Culture",
            "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/description"
        },
        {
            "extension": [
                {
                    "id": "2572130057",
                    "valueString": "Service Connected",
                    "url": "custom-attribute-name"
                },
                {
                    "valueInteger": 1,
                    "url": "custom-attribute-value"
                }
            ],
            "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/custom-attribute"
        },
        {
            "extension": [
                {
                    "id": "2572130093",
                    "valueString": "Special Authority",
                    "url": "custom-attribute-name"
                },
                {
                    "valueInteger": 0,
                    "url": "custom-attribute-value"
                }
            ],
            "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/custom-attribute"
        },
        {
            "extension": [
                {
                    "valueCodeableConcept": {
                        "coding": [
                            {
                                "extension": [
                                    {
                                        "valueCoding": {
                                            "system": "https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/codeSet/14002",
                                            "code": "615214"
                                        },
                                        "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/bill-code-schedule"
                                    }
                                ],
                                "system": "https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/CodeSystem/BillCodes-CPT",
                                "code": "87075"
                            }
                        ]
                    },
                    "url": "code"
                },
                {
                    "valueUnsignedInt": 1,
                    "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/priority"
                }
            ],
            "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/procedure-code"
        },
        {
            "extension": [
                {
                    "valueCodeableConcept": {
                        "coding": [
                            {
                                "extension": [
                                    {
                                        "valueCoding": {
                                            "system": "https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/codeSet/14002",
                                            "code": "3692"
                                        },
                                        "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/bill-code-schedule"
                                    }
                                ],
                                "system": "https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/CodeSystem/BillCodes-MODIFIER",
                                "code": "32"
                            }
                        ]
                    },
                    "url": "code"
                },
                {
                    "valueUnsignedInt": 1,
                    "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/priority"
                }
            ],
            "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/modifier-code"
        },
        {
            "extension": [
                {
                    "valueCodeableConcept": {
                        "coding": [
                            {
                                "extension": [
                                    {
                                        "valueCoding": {
                                            "system": "https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/codeSet/14002",
                                            "code": "615216"
                                        },
                                        "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/bill-code-schedule"
                                    }
                                ],
                                "system": "https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/CodeSystem/BillCodes-PROCCODE",
                                "code": "0N2"
                            }
                        ]
                    },
                    "url": "code"
                },
                {
                    "valueUnsignedInt": 1,
                    "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/priority"
                }
            ],
            "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/procedure-code"
        },
        {
            "valueCodeableConcept": {
                "coding": [
                    {
                        "extension": [
                            {
                                "valueCoding": {
                                    "system": "https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/codeSet/14002",
                                    "code": "615217"
                                },
                                "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/bill-code-schedule"
                            }
                        ],
                        "system": "https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/CodeSystem/BillCodes-REVENUE",
                        "code": "0302"
                    }
                ]
            },
            "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/revenue-code"
        }
    ],
    "status": "billable",
    "code": {
        "coding": [
            {
                "extension": [
                    {
                        "valueCoding": {
                            "system": "https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/codeSet/14002",
                            "code": "667687"
                        },
                        "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/bill-code-schedule"
                    }
                ],
                "system": "https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/CodeSystem/BillCodes-CDM_SCHED",
                "code": "1010222"
            }
        ]
    },
    "subject": {
        "reference": "Patient/12768550"
    },
    "context": {
        "reference": "Encounter/97965088"
    },
    "occurrenceDateTime": "2022-01-21T13:54:00Z",
    "performer": [
        {
            "function": {
                "coding": [
                    {
                        "system": "http://terminology.hl7.org/CodeSystem/v2-0912",
                        "code": "OP",
                        "display": "Ordering Provider"
                    }
                ],
                "text": "Ordering Provider"
            },
            "actor": {
                "reference": "Practitioner/639925"
            }
        },
        {
            "function": {
                "coding": [
                    {
                        "system": "http://terminology.hl7.org/CodeSystem/v3-ParticipationType",
                        "code": "VRF",
                        "display": "verifier"
                    }
                ],
                "text": "verifier"
            },
            "actor": {
                "reference": "Practitioner/1293126"
            }
        }
    ],
    "requestingOrganization": {
        "reference": "Organization/675844"
    },
    "quantity": {
        "value": 1
    },
    "enterer": {
        "reference": "Practitioner/12754008"
    },
    "enteredDate": "2022-01-21T14:02:16Z",
    "reason": [
        {
            "extension": [
                {
                    "valueUnsignedInt": 1,
                    "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/priority"
                }
            ],
            "coding": [
                {
                    "extension": [
                        {
                            "valueCoding": {
                                "system": "https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/codeSet/14002",
                                "code": "3693"
                            },
                            "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/bill-code-schedule"
                        }
                    ],
                    "system": "https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/CodeSystem/BillCodes-ICD",
                    "code": "R51"
                }
            ]
        },
        {
            "extension": [
                {
                    "valueUnsignedInt": 2,
                    "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/priority"
                }
            ],
            "coding": [
                {
                    "extension": [
                        {
                            "valueCoding": {
                                "system": "https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/codeSet/14002",
                                "code": "3693"
                            },
                            "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/bill-code-schedule"
                        }
                    ],
                    "system": "https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/CodeSystem/BillCodes-ICD",
                    "code": "Y92.321"
                }
            ]
        }
    ],
    "account": [
        {
            "reference": "Account/F61865237"
        },
        {
            "reference": "Account/C101453424"
        },
        {
            "reference": "Account/C101453425"
        }
    ]
}`

Comment (krvicky31) [reply lib/resources/example_json/r4_examples_charge_item.rb:240]: Addressed here https://github.com/cerner/fhir.cerner.com/pull/790/files#diff-5cd46ad5f0af8a24217f67b656e77ce3624d6cefec3a18eddf7adc12bd47d35fR8

**Review by alexbeckner1 — APPROVED** [review]

**Comment by krvicky31** [comment]
@SantanuBanerjee Having some issue in login, he has reviewed the changes and approved.

**Review by ashu-bdvt — APPROVED** [review]
Approving changes. Just update the PR title to "R4 ChargeItem: Supporting Information field changes"

**Comment by krvicky31** [comment]
> Approving changes. Just update the PR title to "R4 ChargeItem: Supporting Information field changes"

Changed the title as suggested.

**Comment by NimishaMN** [comment]
Attaching Screenshot after publishing
Screenshot 2022-01-27 at 6 10 24 PM
Screenshot 2022-01-27 at 6 09 48 PM

**Pull request merged**